### PR TITLE
A bunch of random fixes

### PR DESCRIPTION
**Pull request by ghost**
- PercentHealth now no longer predicts "negative" health (Oopsie)
- End/Nether modules now listen on highest priority for better compatibility with multiworld plugins
- End/Nether modules now check if getTo() world is null
- Accepting a team request whitelists all team members

### CODE REVIEW

File: src/main/java/gg/uhc/uhc/modules/portals/EndModule.java [hunk 1]
```diff
@@ -86,17 +88,19 @@ public void onDisable() {
         }
     }
 
-    @EventHandler(ignoreCancelled = true)
+    @EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
```
Comment (Eluinhost) [inline src/main/java/gg/uhc/uhc/modules/portals/EndModule.java]: I don't normally like using HIGHEST priority when were are potentially using setCancelled due to it making it harder to override in another plugin. Is there a reason for using HIGHEST specifically and not just HIGH to go past the regular NORMAL?

File: src/main/java/gg/uhc/uhc/modules/team/requests/RequestManager.java [hunk 2]
```diff
@@ -138,6 +138,7 @@ public boolean finalizeRequest(int id, AcceptState accepted) {
 
             for (OfflinePlayer p : players) {
                 team.addPlayer(p);
+                p.setWhitelisted(true);
```
Comment (Eluinhost) [inline src/main/java/gg/uhc/uhc/modules/team/requests/RequestManager.java]: I don't like the idea of a side effect of whitelisting a team request member, especially if it isn't documented. Can this be documented and added as either 1) a config option or 2) a flag on accepting the request to whitelist/not whitelist depending on what makes more sense logically

**Comment by Eluinhost** [comment]
I've added a couple of notes. Also this should really be in 3 separate commits

File: src/main/java/gg/uhc/uhc/modules/portals/EndModule.java (hunk 1 above)
Comment (ghost) [inline src/main/java/gg/uhc/uhc/modules/portals/EndModule.java]: Good argument, a well behaved multiworld plugin should be listening on LOW/LOWEST. Maybe it's possible to dynamically change this based on configuration and manually registering into the HandlerList, but for now I'll just change this to HIGH.

File: src/main/java/gg/uhc/uhc/modules/team/requests/RequestManager.java (hunk 2 above)
Comment (ghost) [inline src/main/java/gg/uhc/uhc/modules/team/requests/RequestManager.java]: A flag on the command would need large refactoriing due to the structure of the request manager. I'll go with the configuration option & adding documention soon.

**Pull request merged**